### ISSUE TITLE
August Lock Pro with Connect and Doorsense

### ISSUE THREAD

**Issue by dscottbuch** [mlamoure/Indigo-August-Home#7]
Not sure if this is supposed to be supported, or not.  I have the Lock Pro with Connect and have Doorsense installed and calibrated but I only get the option to create and Lock Pro without Doorsense in the New Device interface.  Occasionally the device would show up in the Pro with Doorsense selection but when I choose it, it didn't work.

Thanks,
Scott

**Comment by mlamoure** [comment]
Your configuration is supposed to be supported.  You need to first create the lock device, then create the DoorSense.  When creating the DoorSense you need to choose the lock associated with the door.  

You said that occasionally the lock device would show up but it did not work, can you show any error that occurs?

**Comment by dscottbuch** [comment]
Thanks.  I didn’t understand the ’Sense’ was created as a separate sensor device.  I appeared in the choice when I had created the Door Lock.  You might consider changing the name of the device type to ‘Door Sense for August Lock Pro” to help clarify that.

Thanks again, this saved by bacon as I mis-understood the August marketing material and thought that with the ‘Connect’ device I could use it in HomeKit even if I was not within BLE range of an Apple hub.  I have two doors and can’t have my hub in range of both.  Fortunately I’m running Indigo with HKB.

Again, Thanks!

Scott Buchanan

30475 SW Buckhaven Rd.
Hillsboro,  OR  97123
